api/institutions/serializers.py

```diff
@@ -201,13 +201,38 @@ def get_absolute_url(self, obj):
             },
         )
 
+class UniqueDeptIDField(IDField):
```

Comment:
Sorry this will take longer but what you really should do is make this a class of something like `CompoundUniqueDeptIDField` and make `UniqueDeptIDField` a subclass of that. If you do, remember to put `CompoundUniqueDeptIDField` in a ` api/institutions/...` file path so people can find it to sub-class. 

If you don't, I'll approve this anyway and we'll generalize this when we get multiple case.

If you REALLY want to impress me you'll write unit tests for this field object in isolation without any serializer info. I think we do that one or two places for custom fields, but that's more work for a very small custom field.